src/simulate.rs

```diff
@@ -60,3 +68,49 @@ impl proto::simulate_contract_call_service_server::SimulateContractCallService f
 pub fn get_string_encoded_call(request: ScheduleRequest) -> Result<String, Error> {
     get_encoded_call(request).map(|b| bytes_to_hex_str(&b))
 }
+
+pub fn validate_simulate_tls_config(config: &StewardConfig) {
+    if config.simulate.server_cert_path.is_empty() {
+        panic!("Simulate server cert key is not set");
```

Comment:
This and `validate_tenderly_config` are only called from the simulate command handler, which handles config/validation error by either panicking or shutting down the process (need to revisit this second one because it doesn't gracefully shut down) because the `abscissa_core::run` call that wraps that closure they're in doesn't allow graceful error handling